.github/workflows/ci.yml

```diff
@@ -59,12 +57,22 @@ jobs:
         if: always()
         with:
           junit_files: tests_xml/tests.xml
-          check_name: "Unit Test Results Python ${{ matrix.python-version }}"
+          check_name: "Core / Unit Test Results (${{ matrix.python-version }})"
```

Comment:
I am not sure to understand now what is the difference between the test

`GitHub CI / Core / Unit Tests`

and 

`GitHub CI / Core / Unit Test Results`

Other reviewers might also be confused to see these separate tests.

Reply:
I named them similar on purpose because one is simply the result report of the other. I agree that it is a bit confusing that the report appears as additional "Check". I can disable that. However, the report and check also include Annotations of the failing tests, see https://github.com/CLIMADA-project/climada_python/actions/runs/8817066788 for an example. These allow to quickly hop to the failing code and also appear in the PR "Changes" overview. As far as I can see, Annotations are only available if we keep the Check. See [the docs](https://github.com/marketplace/actions/publish-test-results#commit-and-pull-request-checks) of the associated GitHub action.

If we disable the additional check, one will have to look into the [full test report](https://github.com/CLIMADA-project/climada_python/actions/runs/8817066788/job/24202578024) to find out which code is failing. That is likely more confusing.

If it's just an issue with naming, I welcome suggestions.

Reply:
I don't really see how "Unit Test Summary Report" and "Unit Test Detailed Report" are more clear than "Unit Tests" and "Unit Test Results" 😅 How about "Unit Testing" and "Unit Test Report"?

Reply:
Well, the point is that both cases are reports of test results, just in different formats. So, neither the words "report", "results" or "test" help differentiate both cases. 

Anyway, most people will probably be scared by the test reports now that we have 18 of them. So naming will not change much.

Reply:
> Well, the point is that both cases are reports of test results, just in different formats. 

But the thing you wanted to call "Detailed Report" is actually the output of the entire testing pipeline. It contains much more than a test result or report or whatever you want to call it 😅

Reply:
> How about "Unit Testing" and "Unit Test Report"?

Do we go with this or do we leave it as is?

Reply:
As you prefer. I think `Unit Testing` and `Unit Test Report` are not super clear, but it probably does not matter. @emanuel-schmid may have a better idea, otherwise, let's keep it as you propose.

Reply:
We now have `Unit Test Pipeline` and `Unit Test Report`